config/config.go

```diff
@@ -442,8 +446,9 @@ func DefaultRPCConfig() *RPCConfig {
 		TimeoutBroadcastTxCommit:  10 * time.Second,
 		WebSocketWriteBufferSize:  defaultSubscriptionBufferSize,
 
-		MaxBodyBytes:   int64(1000000), // 1MB
-		MaxHeaderBytes: 1 << 20,        // same as the net/http default
+		MaxRequestBatchSize: 0,              // no maximum requests in a batch request
```

Comment:
is this a good default?

Reply:
I had the default initially set to 10, but revert to 0 because @cason in the issue for this PR mentioned we should keep it at 0 for backwards compatibility (no restrictions). 

But that might not do much good, probably we can enforce a maximum and people will have to change to 0 if they don't want the limit. In my experience, most operators just leave the default values and that might not be beneficial. 

Should I change the default value to something like 10 ? Usually there's another limit for max body request bytes but that's also very high by default so it's hard to trigger. Wdyt?

Reply:
Agree we should constrain the value by default, say to `10`. Since we're introducing a new parameter to configure this behavior, operators are free to mimic the old behavior by setting `0` if they want.

Reply:
OK, I'll revert back to 10, makes sense.

Reply:
reverted